includes/init_includes/init_customer_auth.php

```diff
@@ -79,27 +79,34 @@
  * 2 = customer authorization pending can browse but no prices
  */
 switch (true) {
+  /**
+   * bypass redirects for these scripts, to processing regardless of store mode or cust auth mode
+   */
+  case (preg_match('/ipn_main_handler\.php$/', $_SERVER['SCRIPT_NAME'])):

```

Comment:
Just thinking about the case where someone adds a file with the same name in a subdirectory. More of an edge case, but thinking about 3rd party code adding a page, module, or template. Would it be desirable for such a modification to be always accessible irregardless of the customer approval setting?